docs/sources/dashboards/dashboard-public/index.md

```diff
@@ -40,34 +40,36 @@ If you are using Docker, use an environment variable to enable public dashboards
 
 > **Note:** For Grafana Cloud, you will need to contact support to have the feature enabled.
 
-#### Make a dashboard public
+## Make a dashboard public
 
-- Click on the sharing icon to the right of the dashboard title.
-- Click on the **Public dashboard** tab.
-- Acknowledge the implications of making the dashboard public by checking all the checkboxes.
-- Click **Generate public URL** to make the dashboard public and make your link live.
-- Copy the public dashboard link if you'd like to share it. You can always come back later for it.
+1. Click on the sharing icon to the right of the dashboard title.
+1. Click on the **Public dashboard** tab.
```

Comment:
```suggestion
1. Click the sharing icon to the right of the dashboard title.
1. Click the **Public dashboard** tab.
```

Reply:
I plan to circle back and do a copy edit on a different PR, but just wanted to get the formatting stuff fixed first.